src/VisualStudio/Core/Def/Implementation/FindReferences/Filters/EntireSolutionAndExternalFilterFactory.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.Internal.VisualStudio.Shell.ErrorList;
+using Microsoft.VisualStudio.Shell.TableControl;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.FindReferences.Filters
+{
+    [Export(typeof(IScopeFilterFactory))]
+    [TableManagerIdentifier("FindAllReferences*")]
```

Comment:
what does the * mean?

Reply:
Just a wildcard. This is what we use elsewhere for RichNav, and all of the filters in the VS repo use wildcards everywhere too. Guessing it has to do with allowing multiple find references windows.